exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterMetrics.java

```diff
@@ -92,6 +92,10 @@ private Meter meter() {
         .get("io.opentelemetry.exporters." + exporterName + "-" + transportName);
   }
 
+  private static boolean isNoop(LongCounter counter) {
```

Comment:
Let's add a note in here that we may want to look for a more reliable signal.

Reply:
done.